### PR TITLE
Update havp directories for 2.2.2 

### PR DESCRIPTION
**Pull request by ncolunga**
This changes fix HAVP on pfsense 2.2.2 as related on https://forum.pfsense.org/index.php?topic=90706.0

### CODE REVIEW

**Comment by BBcan177** [comment]
Just to let you know that It looks like Havp will be only available for 2.2.2 and below. 

[Github LINK](https://github.com/pfsense/pfsense-packages/commit/31372afe3ed3fb8691aa726adb75cd56c5b1f290)

**Comment by phil-davis** [comment]
Just to clarify, that change makes it available up to 2.2.999 - effectively 2.2.* - so for all 2.2.* bugfix releases - 2.2.3...

**Comment by ncolunga** [comment]
I'm not sure if it's a good idea to increase package to 2.3.*

File: config/havp/havp.inc [hunk 1]
```diff
@@ -76,7 +76,7 @@ define('HVDEF_PROXYPORT',	            '8080');
 define('HVDEF_MAXSCANSIZE',             '5000000'); # [bytes] ! do not enter 0 or big size !
 define('HVDEF_MAXARCSCANSIZE',          '5000000'); # [bytes] ! do not enter 0 or big size !
 define('HVDEF_PID_FILE',                '/var/run/havp.pid');
-define('HVDEF_WORK_DIR',                '/usr/local/etc/havp');
+define('HVDEF_WORK_DIR',                '/usr/pbi/havp-amd64/local/etc');
```
Comment (rbgarga) [inline config/havp/havp.inc]: You need to check pfSense version and also system arch (amd64 or i386) before set it. Take a look at an example at https://github.com/pfsense/pfsense-packages/blob/master/config/snort/snort_defs.inc#L38

**Comment by ncolunga** [comment]
Version and arch checks done.

**Comment by cbuechler** [comment]
Merged, thanks!

**Pull request closed without merging**